plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -203,6 +204,19 @@ public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsPr
         return this;
     }
 
+    public Optional<String> getAwsCredentialsProviderConf()
+    {
+        return awsCredentialsProviderConf;
+    }
+
+    @Config("hive.metastore.glue.aws-credentials-provider-conf")
+    @ConfigDescription("Configuration details/file accepted by custom AwsCredentialsProvider class ")
+    public GlueHiveMetastoreConfig setAwsCredentialsProviderConf(String awsCredentialsProviderConf)
```

Comment:
Please add a test to `io.trino.plugin.hive.TestHivePlugin` where you verify that no one can set `hive.metastore.glue.aws-credentials-provider-conf` when `hive.metastore.glue.aws-credentials-provider` is not set.

Reply:
Updated the PR